src/ucs/arch/ppc64/cpu.h

```diff
@@ -41,12 +41,12 @@ BEGIN_C_DECLS
 
 static inline uint64_t ucs_arch_read_hres_clock()
 {
-#ifndef HAVE_SYS_PLATFORM_PPC_H
+#if HAVE_DECL___PPC_GET_TIMEBASE && defined(HAVE___PPC_GET_TIMEBASE)
```

Comment:
WDYT to use one macro?
```
AC_CHECK_DECLS([__ppc_get_timebase_freq], 
               [AC_CHECK_FUNCS([__ppc_get_timebase_freq])], [],
               [[#include <sys/platform/ppc.h>]])

```

Then:
```
static inline uint64_t ucs_arch_read_hres_clock()
{
#ifdef HAVE___PPC_GET_TIMEBASE
    return __ppc_get_timebase();
#else
```

Reply:
done.
also, found that `AC_CHECK_FUNCS` couldn't be used for `__ppc_get_timebase()` - it reports that it is undefined.
using `AC_COMPILE_IFELSE` which allows including header file helps to fix it